docs/index.asciidoc

```diff
@@ -75,7 +75,7 @@ include::static/shutdown.asciidoc[]
 include::static/upgrading.asciidoc[]
 
 // Configuring Logstash
-include::static/configuration.asciidoc[]
+include::static/pipeline-configuration.asciidoc[]
 
```

Comment:
Note to self @karenzone: Verified that all existing content is accounted for in source file restructure.